Directory.Build.props

```diff
@@ -90,6 +90,8 @@
     <MicrosoftIdentityAbstractions>6.0.0</MicrosoftIdentityAbstractions>
     <NetNineRuntimeVersion> 9.0.0-preview.4.24266.19</NetNineRuntimeVersion>
     <AspNetCoreNineRuntimeVersion> 9.0.0-preview.4.24267.6</AspNetCoreNineRuntimeVersion>
+    <!--CVE-2024-30105-->
+    <SystemJsonTextVersion>8.0.4</SystemJsonTextVersion>
```

Comment:
these are for .NET 9, which isn't part of the CVE.

Reply:
We'd probably want to move to preview6 anyway
https://www.nuget.org/packages/System.Security.Cryptography.Xml/9.0.0-preview.6.24327.7